worker/jobs/bulkDeleteSubjectsJob.js

```diff
@@ -119,9 +119,9 @@ module.exports = (job, done) => {
           dbStartTime,
           dbEndTime,
         });
+        jobLog(jobStartTime, job);
```

Comment:
no - it's a separate feature toggle, see my prev comment

Reply:
done.